crates/red_knot/src/module.rs

```diff
@@ -177,15 +179,6 @@ impl std::fmt::Display for ModuleName {
     }
 }
 
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
```

Comment:
I removed this declaration and instead use the one from` red_knot_python_semantic` to silence `cargo shear` that the `red_knot_python_semantic` crate is unused (which is true, but not for long)